docs/NNICTLDOC.md

```diff
@@ -49,7 +49,8 @@ nnictl webui
      
       | Name, shorthand | Required|Default | Description |
       | ------ | ------ | ------ |------ |
-    | --experiment, -e|  False| |ID of the experiment you want to resume|
+    | --port, -p|  False| |Rest port of the experiment you want to resume|
+    | --id, -i|  True| |The id of the experiment you want to resume|
```

Comment:
it would be better to put `--id` before `--port`. The `Required` field of `--id` could be False, which means resuming the current experiment.